README.md

```diff
@@ -91,16 +219,627 @@ impl DeserializeFromValue<MyError> for Name {
 }
 ```
 
-### Using macros
+### Supported features
 
-```rust,ignore
-#[derive(DeserializeFromValue)]
-#[deserr(error = MyError)]
-struct User {
-	name: Name,
+#### `rename_all`
+
+Rename all the fields of the struct according to the given case convention.
+The possible values are `lowercase`, `camelCase`.
+If you need more case conventions, please open an issue; adding more is trivial..
+
+```rust
+use deserr::{DeserializeFromValue, deserialize};
+use deserr::serde_json::JsonError;
+use serde_json::json;
+
+#[derive(DeserializeFromValue, Debug, PartialEq, Eq)]
+#[deserr(rename_all = camelCase)]
+struct Search {
+    query: String,
+    attributes_to_retrieve: Vec<String>,
+}
+
+let data = deserialize::<Search, _, JsonError>(
+    json!({ "query": "doggo", "attributesToRetrieve": ["age", "name"] }),
+)
+.unwrap();
+
+assert_eq!(data, Search {
+    query: String::from("doggo"),
+    attributes_to_retrieve: vec![String::from("age"), String::from("name")],
+});
+```
+
+#### `deny_unknown_fields`
+
+Throw an error when encountering unknown fields.
+When this attribute is absent, unknown fields are ignored by default.
+
+```rust
+use deserr::{DeserializeFromValue, deserialize};
+use deserr::serde_json::JsonError;
+use serde_json::json;
+
+#[derive(DeserializeFromValue, Debug)]
+#[deserr(deny_unknown_fields)]
+struct Search {
+    query: String,
+}
+
+let err = deserialize::<Search, _, JsonError>(
+    json!({ "query": "doggo", "doggo": "bork" }),
+)
+.unwrap_err();
+
+assert_eq!(err.to_string(), "Json deserialize error: unknown field `doggo`, expected one of `query` at ``.");
```

Comment:
Not yet, because I didn't merge the meilisearch json error in deserr.
But I'll do it before the next release and update the README with the new messages :+1: